### ISSUE TITLE
Redirect links to content that will now live on teach.mozilla.org

### ISSUE THREAD

**Issue by hannahkane** [mozilla/teach.webmaker.org#266]
Let's make a list of urls that need to be redirected once the new site is live:
 * webmaker.org/resources should go to ~~teach.mozilla.org/web-literacy~~ teach.mozilla.org/teach-like-mozilla/web-literacy/
 * webmaker.org/literacy should go ~~to teach.mozilla.org/web-literacy~~ teach.mozilla.org/teach-like-mozilla/web-literacy/
 * webmaker.org/party should go to teach.mozilla.org/events
 * party.webmaker.org should go to teach.mozilla.org/events
 * https://webmaker.org/en-US/mentor should go to teach.mozilla.org

/cc @amirad @dajbelshaw @LauraHilliger @ldecoursy @chrislarry33 @thornet @secretrobotron

**Comment by hannahkane** [comment]
Wondering if we need some sort of contextual message for users who arrive this way. e.g. "Teach.mozilla.org is the new home of Maker Party!"

**Comment by dajbelshaw** [comment]
@hannahkane:

Agree that:

* webmaker.org/resources should go to learn.mozilla.org/web-literacy

However, I would suggest that

* webmaker.org/literacy should go to **teach**.mozilla.org/web-literacy

(or whatever we're designing in #52)

**Comment by hannahkane** [comment]
Sorry, @dajbelshaw, I meant for all of the links to go to teach.mozilla.org/*. Have updated the comment to reflect that.

I was assuming both webmaker.org/resources and webmaker.org/literacy would go to the same page on the new site, since we're essentially combining that content into a single page, as per #52. 

Let me know if that doesn't make sense.

**Comment by dajbelshaw** [comment]
Ah... that makes much more sense. Thanks @hannahkane :)

**Comment by LauraHilliger** [comment]
maybe this spreadsheet @secretrobotron and @thornet used for the interim IA is useful? If not, disregard:

https://docs.google.com/a/mozillafoundation.org/spreadsheets/d/1wpqM0zQs8BRICLauEn8hcY4T_2FiE_juhkjkNgw9Tqw/edit#gid=0

**Comment by hannahkane** [comment]
Thanks, @LauraHilliger, that's super helpful. 

Looks like we've covered most of what's on the spreadsheet, with a couple exceptions: 
 * I don't think /explore needs a re-direct - it can continue to go to the gallery, which can continue to live on webmaker.org
 *  I do have a question about what to do with https://www.webmaker.org/make-your-own. @LauraHilliger and @thornet - is there a new page on the github curriculum site that you'd like to have that go to?

**Comment by chrislarry33** [comment]
Might suggest we just take down "make your own" on webmaker and not have a redirect at this point.

**Comment by thornet** [comment]
Would it be worth adding a "make your own" button the new curriculum page, for folks looking for those templates? I know that won't be a long-term solution, but at least that resource doesn't completely disappear.

**Comment by hannahkane** [comment]
Just a note: we don't actually have to do anything with that page. We could keep it on Webmaker, and keep the /make-your-own shortcut, so anyone who had it bookmarked could still find it. I think it's more an issue of do we want to be promoting Thimble makes, or do we want to provide new github-flavored instructions for creating curriculum.

**Comment by mmmavis** [comment]
Currently blocked by https://github.com/mozilla/teach.webmaker.org/issues/828

**Comment by mmmavis** [comment]
> * party.webmaker.org should go to teach.mozilla.org/events

https://github.com/mozilla/teach.webmaker.org/issues/828 has been merged.  We'll have to wait till it's live on PROD before we can work on this ticket.

**Comment by asdofindia** [comment]
http://webmaker.org/teach should go to teach.mozilla.org I guess. Or, https://support.mozilla.org/en-US/kb/top-tips-creating-great-teaching-kits should be updated

**Comment by LauraHilliger** [comment]
@asdofindia thanks for pointing out the support article. I went ahead and updated https://support.mozilla.org/en-US/kb/top-tips-creating-great-teaching-kits

**Comment by mmmavis** [comment]
@jbuck could you help with this? I've also created a separate ticket on the DevOps repo https://github.com/MozillaFoundation/mofo-devops/issues/69

**Comment by jbuck** [comment]
@mmmavis Why not put the redirects in the code for webmaker.org?

**Comment by mmmavis** [comment]
@jbuck I can submit a PR for pages on `webmaker.org` but our plan is to kill the Maker Party site (https://party.webmaker.org/) and I guess that part needs your help (including redirecting it to teach.mozilla.org/events) :hamster:

**Comment by jbuck** [comment]
Sorry, I thought it was all webmaker.org :blush: . Is it alright if I make that redirect from https://party.webmaker.org to https://teach.mozilla.org/events now? Or should it be done sometime later?

**Comment by mmmavis** [comment]
hey @jbuck I have a [PR open on webmaker.org](https://github.com/mozilla/webmaker.org/pull/1411) to handle some page redirects.  Other than that, please make https://party.webmaker.org redirected to https://teach.mozilla.org/events when you get a chance.  Thanksssss! :bowtie:

**Comment by mmmavis** [comment]
Everything worked except `webmaker.org/literacy`(without trailing slash) doesn't get redirected properly.  I've open [another PR on webmaker.org repo](https://github.com/mozilla/webmaker.org/pull/1413)

**Comment by jbuck** [comment]
@mmmavis I've fixed that redirect so /literacy works correctly, but I think we should back out this redirect. @bobchao, one of our localizers working on the zh-TW version of webmaker.org, pointed out that redirecting to teach.mozilla.org means no localized content.

/cc @hannahkane @simonwex

**Comment by thisandagain** [comment]
Moving the web literacy / teaching resources over to the teach site is critical. Rather than delay a necessary change, I recommend posting a note to the community on Transifex to explain the transition and let them know how to engage / participate with the new site.

**Comment by hannahkane** [comment]
agreed. 

Will file a ticket to port the Literacy page translations specifically, and will continue the ongoing discussion of globalization happening in #916.

Closing this for now.

**Comment by bobchao** [comment]
Hi 

> Rather than delay a necessary change, I recommend posting a note to the community on Transifex to explain the transition and let them know how to engage / participate with the new site.

It will be even better if we do this (the announcement) before the change :) At lease I can back up the HTML file with a single Save As... instead of clone the code and build an old website myself.

Thanks
